aws/table_aws_backup_plan.go

```diff
@@ -0,0 +1,158 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/backup"
+
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBackupPlan(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_backup_plan",
+		Description: "AWS Backup Plan",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AnyColumn([]string{"backup_plan_id"}),
+			ShouldIgnoreError: isNotFoundError([]string{"InvalidParameterValue"}),
+			Hydrate:           getAwsBackupPlan,
+		},
+		GetMatrixItem: BuildRegionList,
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBackupPlans,
+		},
+		Columns: []*plugin.Column{
```

Comment:
Use `awsRegionalColumns` for AWS common columns, (i.e. account_id, partition, region)